sites/friday/src/get-started/laravel/deploy/configure.md

```diff
@@ -0,0 +1,301 @@
+
+Each service entry has a name (`db` and `cache` in the example)
+and a `type` that specifies the service and version to use.
+Services that store persistent data have a `disk` key, to specify the amount of storage.
+
+```yaml {configFile="app"}
+# The services of the project.
+#
+# Each service listed will be deployed
+# to power your Upsun project.
+# More information: https://docs.deployfriday.net/add-services.html
+# Full list of available services: https://docs.deployfriday.net/add-services.html#available-services
+services:
+  mysql:
+    type: mysql:10.6 # All available versions are: 10.6, 10.5, 10.4, 10.3
+
+  redis:
+    type: redis:7.0 # All available versions are: 7.0, 6.2
+
+```
+
+## Define routes
+
+All HTTP requests sent to your app are controlled through the routing and caching
+you define in the `routes` top-level key of the `{{< vendor/configfile "app" >}}` file.
+
+The two most important options are the main route and its caching rules.
+A route can have a placeholder of `{default}`,
+which is replaced by your domain name in production and environment-specific names for your preview environments.
+The main route has an `upstream`, which is the name of the app container to forward requests to.
+
+You can enable [HTTP cache](/define-routes/cache.md).
+The router includes a basic HTTP cache.
+By default, HTTP caches includes all cookies in the cache key.
+So any cookies that you have bust the cache.
+The `cookies` key allows you to select which cookies should matter for the cache.
+Generally, you want the user session cookie, which is included in the example for Laravel.
+You may need to add other cookies depending on what additional modules you have installed.
+
+You can also set up routes as [HTTP redirects](/define-routes/redirects.md).
+In the following example, all requests to `www.{default}` are redirected to the equivalent URL without `www`.
+HTTP requests are automatically redirected to HTTPS.
+
+If you don't include a `{{< vendor/configfile "routes" >}}` file, a single default route is used.
+This is equivalent to the following:
+
+```yaml {configFile="app"}
+https://{default}/:
+  type: upstream
+  upstream: <APP_NAME>:http
+```
+
+Where `<APP_NAME>` is the `name` you've defined in your [app configuration](#configure-apps-in-platformappyaml).
+
+The following example presents a complete definition of a main route for a Laravel app:
+
+```yaml {configFile="app"}
+# The routes of the project.
+#
+# Each route describes how an incoming URL is going
+# to be processed by Upsun.
+# More information: https://docs.deployfriday.net/define-routes.html
+routes:
+  "https://{default}/":
+    type: upstream
+    upstream: "my-laravel-app:http"
+  # A basic redirect definition
+  # More information: https://docs.deployfriday.net/define-routes.html#basic-redirect-definition
+  "https://www.{default}":
+    type: redirect
+    to: "https://{default}/"
+
+```
+
+
```

Comment:
now that we've had them generate/create their configuration files, we need to make sure to remind them to add & commit them to their repository